src/api.ts

```diff
@@ -1,9 +1,8 @@
 import { Context } from "./context";
-import { BadRequestError, InternalServerError } from "./error";
 import { ExaMetadata } from "./exasolSchema";
 import { Parameter } from "./parameters";
 
-export const CURRENT_API_VERSION = "0.1.14";
+export const CURRENT_API_VERSION = "0.1.15";
```

Comment:
duplicated, `package.json` contains this information to is there a way this could be changed to a single source of truth?